### PR TITLE
Add release logs for 1.7.0

### PR DESCRIPTION
**Pull request by jjyao**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by jjyao** [comment]
Don't merge this yet since we may still need to do several cherry picks. But people can review to see if I capture everything and if there is any performance degradation.

File: release/release_logs/1.7.0/stress_tests/dead_actors.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 524, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 561, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 568, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 572, in ray._raylet.execute_task
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "python/ray/_raylet.pyx", line 522, in ray._raylet.execute_task.function_executor
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/function_manager.py", line 579, in actor_method_executor
+[2m[36m(pid=3700, ip=172.31.49.215)[0m     return method(__ray_actor, *args, **kwargs)
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "/home/ray/anaconda3/lib/python3.7/site-packages/ray/util/tracing/tracing_helper.py", line 448, in _resume_span
+[2m[36m(pid=3700, ip=172.31.49.215)[0m     return method(self, *_args, **_kwargs)
+[2m[36m(pid=3700, ip=172.31.49.215)[0m   File "stress_tests/test_dead_actors.py", line 28, in ping
```
Comment (jjyao) [inline release/release_logs/1.7.0/stress_tests/dead_actors.txt:23]: Do we need these?

File: release/release_logs/1.7.0/microbenchmark.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+multi client tasks async per second 38216.25 +- 1653.24
```
Comment (jjyao) [inline release/release_logs/1.7.0/microbenchmark.txt:1]: in 1.6.0 release logs, I saw lines for single client like `single client get calls per second 35445.18 +- 479.15
`,  but it's not here. Is this expected?

File: release/release_logs/1.7.0/benchmarks/many_actors.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Success! Started 100 actors in 2.97153377532959s. (33.65265467625668 actors/s)
```
Comment (jjyao) [inline release/release_logs/1.7.0/benchmarks/many_actors.txt:1]: in 1.6.0 release logs, it is `Success! Started 10000 actors in 30.97493314743042s. (322.84169758828256 actors/s)
`, here we only started 100, is this expected?

Comment (rkooo567) [reply release/release_logs/1.7.0/benchmarks/many_actors.txt:1]: Hmm this is weird. @wuisawesome did we change anything from this test recently? Also @iycheng who worked on the actor scalability

File: release/release_logs/1.7.0/microbenchmark.txt (hunk 2 above)
Comment (rkooo567) [reply release/release_logs/1.7.0/microbenchmark.txt:1]: @mwtian I remember we removed one test in the past. Can you double check if this test is needed?

File: release/release_logs/1.7.0/stress_tests/dead_actors.txt (hunk 1 above)
Comment (rkooo567) [reply release/release_logs/1.7.0/stress_tests/dead_actors.txt:23]: I think these logs are expected, so we don't need to have them here (just the result is enough)

File: release/release_logs/1.7.0/microbenchmark.txt (hunk 2 above)
Comment (mwtian) [reply release/release_logs/1.7.0/microbenchmark.txt:1]: The test results here look unexpected. I think we are missing the first 10 initial entries: [1.6.0](https://github.com/ray-project/ray/blob/d202749c3254d34a71a6883b86c772b850b037d6/release/release_logs/1.6.0/microbenchmark.txt)

File: release/release_logs/1.7.0/benchmarks/many_actors.txt (hunk 3 above)
Comment (wuisawesome) [reply release/release_logs/1.7.0/benchmarks/many_actors.txt:1]: That sounds suspiciously like the smoke test￼

Comment (jjyao) [reply release/release_logs/1.7.0/benchmarks/many_actors.txt:1]: I think I did it wrong here. I should get results from weekly release, not core nightly.

File: release/release_logs/1.7.0/microbenchmark.txt (hunk 2 above)
Comment (jjyao) [reply release/release_logs/1.7.0/microbenchmark.txt:1]: OK, the reason is that we only get the last 50 lines of test output so it's truncated. I'll fix that.

File: release/release_logs/1.7.0/benchmarks/many_actors.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (wuisawesome) [inline release/release_logs/1.7.0/benchmarks/many_actors.txt:1]: this one lgtm

File: release/release_logs/1.7.0/benchmarks/many_nodes.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (wuisawesome) [inline release/release_logs/1.7.0/benchmarks/many_nodes.txt:1]: this one lgtm

File: release/release_logs/1.7.0/benchmarks/many_pgs.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (wuisawesome) [inline release/release_logs/1.7.0/benchmarks/many_pgs.txt:1]: this one lgtm

File: release/release_logs/1.7.0/benchmarks/many_tasks.txt [hunk 7]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (wuisawesome) [inline release/release_logs/1.7.0/benchmarks/many_tasks.txt:1]: this one lgtm

File: release/release_logs/1.7.0/scalability/object_store.txt [hunk 8]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (wuisawesome) [inline release/release_logs/1.7.0/scalability/object_store.txt:1]: this one lgtm

File: release/release_logs/1.7.0/scalability/single_node.txt [hunk 9]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (wuisawesome) [inline release/release_logs/1.7.0/scalability/single_node.txt:1]: this one lgtm

File: release/release_logs/1.7.0/stress_tests/many_tasks.txt [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+  "stage_1_avg_iteration_time": 17.450765538215638,
+  "stage_1_max_iteration_time": 17.627604961395264,
+  "stage_1_min_iteration_time": 17.23277997970581,
+  "stage_2_time": 268.01243686676025,
+  "stage_2_avg_iteration_time": 53.60213441848755,
+  "stage_2_max_iteration_time": 59.097413063049316,
+  "stage_2_min_iteration_time": 48.71518564224243,
+  "stage_3_creation_time": 0.5777060985565186,
+  "stage_3_time": 2066.70570230484,
+  "stage_4_spread": 3.2197082901427945,
```
Comment (wuisawesome) [inline release/release_logs/1.7.0/stress_tests/many_tasks.txt:14]: hmmm this spread has gone up a bit. prob fine, but may be worth a closer look?

Comment (wuisawesome) [reply release/release_logs/1.7.0/stress_tests/many_tasks.txt:14]: According to the comment in the test, the spread used to be significantly worse than this, so it's probably just noise. This should be fine.

**Review by ericl — APPROVED** [review]

File: release/release_logs/1.7.0/stress_tests/placement_group.txt [hunk 11]
```diff
@@ -0,0 +1,9 @@
+{
+  "success": 1,
+  "avg_pg_create_time_ms": 0.9874122837809874,
```
Comment (rkooo567) [inline release/release_logs/1.7.0/stress_tests/placement_group.txt:3]: LGTM

**Pull request merged**